pypeit/tests/test_bitmask.py

```diff
@@ -99,13 +97,13 @@ def test_wrong_bits():
     cosmics_indx[numpy.random.randint(0,high=n,size=100),
                  numpy.random.randint(0,high=n,size=100)] = True
     mask[cosmics_indx] = image_bm.turn_on(mask[cosmics_indx], 'COSMIC')
-
-#    out = image_bm.flagged(mask, 'JUNK')
     
-    with pytest.raises(ValueError):
+    # Fails with all bad flags
+    with pytest.raises(PypeItError):
         out = image_bm.flagged(mask, flag='JUNK')
 
-    with pytest.raises(ValueError):
+    # Fails with mix of good and bad flags
+    with pytest.raises(PypeItError):
```

Comment:
Same comment about the changes to `bitmask.py`.

Reply:
Reverted.